src/main/java/uk/gov/hmcts/reform/sscs/model/LaCaseStatus.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.Locale;
+
+@Getter
+public enum LaCaseStatus {
```

Comment:
Does this have to be LaCaseStatus? Can't we use ListAssistCaseStatus?

Same enum comments as in Dholaram's PR apply.

Reply:
![image](https://user-images.githubusercontent.com/93123095/158450667-f9cc84f9-70cf-4930-b16d-009f68eef733.png)

The mapping name is LaCassStatus
will remove the getByLabel method

Reply:
Its different in different places, confirm with Andrew the correct one?